src/Shared/UnitTests/ObjectModelHelpers.cs

```diff
@@ -671,113 +672,135 @@ internal static ProjectRootElement CreateInMemoryProjectRootElement(string proje
         /// <summary>
         /// Create a project in memory. Load up the given XML.
         /// </summary>
-        /// <param name="xml"></param>
-        /// <returns></returns>
+        /// <param name="xml">the project to be created in string format.</param>
+        /// <returns>Returns created <see cref="Project"/>.</returns>
         internal static Project CreateInMemoryProject(string xml)
         {
-            return CreateInMemoryProject(xml, new ConsoleLogger());
+            return CreateInMemoryProject(xml, new[] { new ConsoleLogger() });
```

Comment:
nit: Doesn't `param` take care of creating the array here?

```suggestion
            return CreateInMemoryProject(xml, new ConsoleLogger());
```